vsintegration/src/FSharp.ProjectSystem.PropertyPages/PropertyPages/TargetFrameworkMoniker.vb

```diff
@@ -64,13 +97,77 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                         supportedFrameworksList.Add(framework)
                     End If
                 Next
-
                 Return supportedFrameworksList.ToArray
             End If
-
             Return prgSupportedFrameworks
         End Function
 
+        Public Shared Function GetDisplayIdFromMoniker(moniker As String) As String
+            If moniker.StartsWith(".NETStandard", StringComparison.OrdinalIgnoreCase) Or
```

Comment:
I'm not a VB expert, but It is worth making a `MonikerStartsWith` function to replace `moniker.StartsWith(".NETStandard", StringComparison.OrdinalIgnoreCase)`

by

`MonikerStartsWith(moniker,".NETStandard")`.

That would clear all the repeated `StringComparison.OrdinalIgnoreCase` and make it easier to read / edit.

The otherthing is `OrElse` operator should be used IIRC, otherwise it doesn't shortcircuit.

Reply:
OrElse I agree, but a naiive wrapper to hide the , StringComparison.OrdinalIgnoreCase) hides an important element of the comparison.  Besides characters are cheap ... costco always has big boxes of them.

Reply:
@KevinRansom I feel the opposite on hiding important information 😄 the important information from my perspective is:

* is there more than one way to compare them, right now I need to scan every calls to make sure it is `OrdinalIgnoreCase`
* doesn't look fun to update it, and add more of them

anyways, was just minor stylistic remark, who maintains the file knows more than I :)

Reply:
@smoothdeveloper, almost the first question I ask when looking at a compare is "How should case and culture be handled, it annoys me when I have to scroll to find out.  I hate that the durned Api's are so wordy and that there are so many different ways of doing it right, but it is not something I want to have to figure out with each comparison.  Regardless of that, I am not thrilled about a naiive wrapper.

@smoothdeveloper TBH ... if someone had said ... the whole structure of this wrong I would have sighed but agreed ... I started out just wanting to fix it without rearranging the structure of it, but the changes grew and grew, now, I am going to see if there is not a better idea on how to do this whole thing.